include/fastdds/dds/topic/ContentFilteredTopic.hpp

```diff
@@ -55,26 +56,70 @@ class ContentFilteredTopic : public TopicDescription
 
 public:
 
-    /**
-     * @brief Destructor
-     */
     RTPS_DllAPI virtual ~ContentFilteredTopic();
```

Comment:
Following #2536, I think the `RTPS_DllAPI` should be removed. However, in order to merge this PR, we cannot do it right now. Maybe issuing another PR with this change and setting the milestone to `2.7.0` is the way to go.

Reply:
I will do another PR removing the dll export.

Reply:
See #2611